internal/federationin/federationin.go

```diff
@@ -48,9 +48,11 @@ var (
 	fetchBatchSize = database.InsertExposuresBatchSize
 )
 
-type fetchFn func(context.Context, *pb.FederationFetchRequest, ...grpc.CallOption) (*pb.FederationFetchResponse, error)
-type insertExposuresFn func(context.Context, []*model.Exposure) error
-type startFederationSyncFn func(context.Context, *model.FederationInQuery, time.Time) (int64, database.FinalizeSyncFn, error)
+type (
```

Comment:
I guess this makes sense, but I've personally never seen it in any Go for types (common on `var` and `const`). Is this common?

Reply:
No, groups with `type` are relatively rare. In this case I felt it looked better, because each of the function types is a single line.